README.md

```diff
@@ -27,19 +27,21 @@ Step 2:Navigate to the root of the directory you cloned and run:
 | `npm run build` | Builds the application as a Web Extension.|
 | `npm start`     | Launches Firefox with the Web Extension.  |
 
+You can also open the `test/index.html` file in your browser to run the automated tests.
 
 ## Localization
 
 Firefox Notes localization is managed via [Pontoon](https://pontoon.mozilla.org/projects/test-pilot-notes/), not direct pull requests to the repository. If you want to fix a typo, add a new language, or simply know more about localization, please get in touch with the [existing localization team](https://pontoon.mozilla.org/teams/) for your language, or Mozilla’s [l10n-drivers](https://wiki.mozilla.org/L10n:Mozilla_Team#Mozilla_Corporation) for guidance.
 
 ## Licenses
 
-[Mozilla Public License Version 2.0](LICENSE)
+* [Mozilla Public License Version 2.0](LICENSE)
 
-[Quill Rich Text Editor License](https://github.com/quilljs/quill/blob/develop/LICENSE)
-[CKEditor Text Editor License](https://github.com/ckeditor/ckeditor5/blob/master/LICENSE.md) used under MPL licence
+* [Quill Rich Text Editor License](https://github.com/quilljs/quill/blob/develop/LICENSE)
```

Comment:
Do we still need the Quill license info? Or wasn't that removed in 1.9.x?

Reply:
We still need it to migrate previous content from an old version of the add-on to the new one. We might remove it a bit later.